pitest-maven-verification/src/test/java/org/pitest/PitMojoIT.java

```diff
@@ -353,17 +354,14 @@ public void shouldCorrectlyHandleOverrides() throws Exception {
         buildFilePath(testDir, "target", "site", "pit-reports").exists());
   }
 
-  @Test
+  @Test(expected = FileNotFoundException.class)
```

Comment:
This test checked if exclusions were read from surefire config, skipping tests only as a way of speeding the test up.

I think it needs to be updated to now ommit the`-DskipTests`, keeping the original assertion. A new test case is needed to ensure that the tests are now skipped when skipTests are set.

Reply:
Okey, now I understand.

I will fix it with new commit :)

Reply:
@hcoles can I ask you for another review :) ?